tests/Basic/Basic.h

```diff
@@ -486,8 +486,9 @@ DLL_API int Function()
 }
 
 // Tests properties
-struct DLL_API TestProperties
+class DLL_API TestProperties
```

Comment:
Please add a few words explaining why this type has to be a class. I just want to be sure there's no hidden bug revealed by this test.